sample-211/src/main/java/com/infullmobile/android/infullmvp/sample/dagger11/sample_mvp_fragment/SampleMvpFragment.kt

```diff
@@ -0,0 +1,19 @@
+package com.infullmobile.android.infullmvp.sample.dagger11.sample_mvp_fragment
+
+import com.infullmobile.android.infullmvp.InFullMvpFragment
+
+import dagger.android.support.AndroidSupportInjection
+import javax.inject.Inject
+
+class SampleMvpFragment : InFullMvpFragment<SampleMvpFragmentPresenter, SampleMvpFragmentVIew>() {
```

Comment:
Your SampleMvpFragmentVIew has capital "i". Beware, git is not case sensitive.

Reply:
Your file system on Mac isn't.

`git config core.ignorecase` is by default set to false.